### PR TITLE
Enhance `Accordion` widget to accept `BackedEnum` for CSS classes.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

**Review by samdark — APPROVED** [review]

File: src/Accordion.php [hunk 1]
```diff
@@ -71,24 +71,21 @@ public function addAttributes(array $values): self
      * Multiple classes can be added by passing them as separate arguments. `null` values are filtered out
      * automatically.
      *
-     * @param string|null ...$value One or more CSS class names to add. Pass `null` to skip adding a class.
+     * @param BackedEnum|string|null ...$values One or more CSS class names to add. Pass `null` to skip adding a class.
      * For example:
      *
      * ```php
-     * $accordion->addClass('custom-class', null, 'another-class');
+     * $accordion->addClass('custom-class', null, 'another-class', BackGroundColor::PRIMARY());
```
Comment (vjik) [inline src/Accordion.php:78]:
```suggestion
     * $accordion->addClass('custom-class', null, 'another-class', BackGroundColor::PRIMARY);
```

File: src/Accordion.php [hunk 2]
```diff
@@ -170,19 +167,19 @@ public function bodyAttributes(array $values): self
      * Multiple classes can be added by passing them as separate arguments. `null` values are filtered out
      * automatically.
      *
-     * @param string|null ...$value One or more CSS class names to set. Pass `null` to skip setting a class.
+     * @param BackedEnum|string|null ...$values One or more CSS class names to set. Pass `null` to skip setting a class.
      * For example:
      *
      * ```php
-     * $accordion->class('custom-class', null, 'another-class');
+     * $accordion->class('custom-class', null, 'another-class', BackGroundColor::PRIMARY());
```
Comment (vjik) [inline src/Accordion.php:174]:
```suggestion
     * $accordion->class('custom-class', null, 'another-class', BackGroundColor::PRIMARY);
```

**Review by vjik — APPROVED** [review]

File: src/Accordion.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Accordion.php]: done.

File: src/Accordion.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Accordion.php]: done.

**Pull request merged**